store/actions/crypto.actions.tsx

```diff
@@ -20,31 +20,43 @@ export const readData = () => {
                 }
             })
             .catch(err => { console.error(err) })
-
     }
 }
 
 export const addCripto = (textInput: string) => {
     return async (dispatch: Dispatch<Action>, getState: () => RootState) => {
+
         const { cryptoList } = getState().cripto
+        const response = await fetch(`https://data.messari.io/api/v1/assets/${textInput.toLowerCase()}/metrics?fields=id,symbol,name,market_data/price_usd,market_data/percent_change_usd_last_24_hours`)
 
-        const newCrypto: Crypto | undefined = cryptos.find(({ name, initials }) => {
-            const addedCrypto: Crypto | undefined = cryptoList.find(({ name, initials }) =>
-                (textInput === name || textInput === initials)
-            )
-            return !addedCrypto && (name === textInput || initials === textInput)
-        })
-        if (newCrypto) {
-            dispatch({
-                type: ADD_CRYPTO,
-                payload: newCrypto
-            })
+        if (!response.ok) {
```

Comment:
[suggestion] This is not wrong, but best practices follow asserting conditionals. Please invert this condition so the positive affirmation comes first